run-tests-with-build-jbpm/pom.xml

```diff
@@ -73,9 +77,29 @@
           <pomIncludes>
             <pomInclude>jbpm/pom.xml</pomInclude>
           </pomIncludes>
-          <goals>
-            <goal>${maven.default.command} -pl !:jbpm-audit,!:jbpm-bpmn2,!:jbpm-case-mgmt,!:jbpm-executor-ejb,!:jbpm-human-task-core,!:jbpm-installer,!:jbpm-persistence-jpa,!:jbpm-services-ejb-impl,!:jbpm-services-ejb-timer,!:jbpm-test,!:jbpm-container-test-suite,!:jbpm-remote-ejb-test-suite</goal>
-          </goals>
+          <pomExcludes>
```

Comment:
Hm, is this actually neede?
In pomIncludes there's just the jbpm/pom.xml, meaning whole jbpm will be built. AFAIK pomExclude does not alter the maven reactor, it's just excluding something that is previously included ...

Reply:
For standardization reasons, instead of ignoring modules in the command line, I decided to ignore them like the other modules do: using <pomExclude> tag.

Reply:
Please double-check that it does what you think it does. IMO since we have just a single pomInclude, being it a root jbpm/pom.xml, the full reactor build is executed, pomExclude does not change anything - once the pom.xml passes includePoms+excludePoms wildcard matching the invoker has no means of altering the scope of reactor - that's why there was this `-pl  !:jbpm-audit`, etc probably.

Reply:
restored the logic with maven.default.command from parent pom.xml in jbpm module. I think that the pomInclude could be altered to jbpm/*/pom.xml and keep the pomExclude tags, but I'm not sure if it would work well for jbpm module.